shared/common-adapters/button.native.js

```diff
@@ -58,30 +58,34 @@ class Button extends Component<void, Props, void> {
 
     // Need this nested view to get around this RN issue: https://github.com/facebook/react-native/issues/1040
     return (
-      <ClickableBox
-        style={borderRadius}
-        onClick={onPress}
-        underlayColor={style.backgroundColor}>
-        <Box style={{...style, ...this.props.style, alignItems: 'center', justifyContent: 'center'}}>
-          <Text type={this.props.small ? 'BodySemibold' : 'BodyBig'} style={{...labelStyle, ...this.props.labelStyle}}>{this.props.label}</Text>
-          {this.props.waiting && <Progress />}
-        </Box>
-      </ClickableBox>
+      <Box style={_containerStyle}>
```

Comment:
for a good before and after go into the dumb sheet on master/this and click and hold on the primary button